### ISSUE TITLE
Script API compatibility files are not found

### ISSUE THREAD

**Issue by frosch123** [OpenTTD/OpenTTD#8618]
## Version of OpenTTD
20210129-master-g48698403a8
linux, self-compiled from checkout

## Expected result
Starting a game with SiliconValley game script should load the 1.3 API compatibility script, and print
"1.3 API compatibility in effect." in the GS log.

## Actual result
The GS log shows "API compatibility script not found".
The script crashes later due to API incompatibility.

## Steps to reproduce
1. There is no system-wide installation of OpenTTD.
2. Start OpenTTD directly from "build" folder.
3. Add "SiliconValley" as GS, and start a game.
4. Check the GS log for one of the two messages above.

**Comment by frosch123** [comment]
I guess cmake needs to copy the scripts to the build folder?

**Comment by TrueBrain** [comment]
Some background, for those who are interested: when I started the CMake branch, the idea was to have the current-working-directory in `bin`, so these files didn't have to be copied and constantly checked for: "are you modified?". `lang` is loaded from where-ever `openttd` is located (inside the `build` folder), which on the targets I was testing worked fine.

Sadly ... it turns out initially my set of targets I tested with was too limited :) On Windows it is pretty common that``openttd.exe` lives in `build/Debug` and `build/Release`, meaning it cannot find `lang` (which is in `build/lang`). As such, the current-working-directory has to be your build-folder for it to work. So this whole idea had to be thrown out of the window, and the content of `bin` has to be copied inside the `build` folder.

I am guessing, but I have not checked the code, that this last part never happened yet. I think I never left that as comment in the PR how I left it, and in normal testing this won't show up ... but who is to blame for not implementing this is not important now, is it? :D

So yeah, sounds like the best solution is to copy the files from the `bin` folder inside the `build` folder via CMake :) Just be careful that changes to those file have to trigger a new copy .. but by now we have enough experience with that, not? (and with "we" I guess I mean @glx22 .. sorry :P)
